src/middleware.js

```diff
@@ -0,0 +1,34 @@
+const http = require('http')
+const https = require('https')
+
+/**
+ * Creates a new instance of the middlware
+ * @return {Function} An express middleware function
+ */
+module.exports = function(server) {
+  return (req, res) => {
+    const exportedValues = {}
+
+    req.pathname = req.path
```

Comment:
@markbrocato I know this module is going to be published, but in order to get this to work. I had to include this line.